src/pages/settings/Profile/LoginField.js

```diff
@@ -86,7 +86,7 @@ class LoginField extends Component {
 
         return (
             <View style={styles.mb6}>
-                <Text style={styles.formLabel}>{this.props.label}</Text>
+                <Text style={[styles.formLabel, !this.props.login.validatedDate && styles.mb0]}>{this.props.label}</Text>
```

Comment:
@thesahindia Phone number has 0 bottom-margin for all the cases - no phone number, unverified, verified.
 
![Screenshot from 2022-02-03 15-45-57](https://user-images.githubusercontent.com/29673073/152346902-5f76b444-325f-45fb-8e4e-056bcb69c3ac.png)

Reply:
@rushatgabhane, verified one should have the bottom-margin.
Here's the screenshot -
<img width="371" alt="Screenshot 2022-02-03 at 6 37 39 PM" src="https://user-images.githubusercontent.com/83179295/152349143-7d6d6cef-dd7f-4f70-8c92-544327c0b263.png">

Reply:
Oh yeah, makes sense

Reply:
@rushatgabhane to clarify, do we need bottom margin in no phone number case?

Reply:
Yes. Bottom margin for no phone number case @thesahindia

Reply:
Done.